Open Judge System/LocalWorker/OJS.LocalWorker/OjsSubmissionProcessingStrategy.cs

```diff
@@ -170,6 +174,29 @@ protected override void ProcessTestsExecutionResult(IExecutionResult<TestResult>
             this.UpdateResults();
         }
 
+        private void AssignWorkerType()
+        {
+            var currentSubmission = this.submissionsData.GetById(this.submissionForProcessing.SubmissionId);
```

Comment:
Don't we have the submission already retrieved somewhere in here?

Reply:
Fixed.